examples/Demo/Server/Pages/_Layout.cshtml

```diff
@@ -32,7 +32,7 @@
     </div>
 
     <script src="_framework/blazor.server.js"></script>
-
+    <script src="https://cdn.jsdelivr.net/npm/sortablejs@1.13.0/Sortable.min.js"></script>
```

Comment:
Why is it not the same version in Server vs Client?

Reply:
Trying to fix an error. Corrected it now